hal/hal.h

```diff
@@ -45,6 +45,13 @@ extern int hal_cpuJump(addr_t addr);
 extern addr_t hal_kernelGetAddress(addr_t addr);
 
 
+/* Function validates and add memory map at hal level: MMU, MPU */
+extern int hal_memAddMap(addr_t start, addr_t end, u32 attr, u32 id);
+
+/* Function prints hal specific memory information eg. region setup */
+extern void hal_memInfo(void);
+
```

Comment:
This function is redundant. All information about maps should be displayed by map command belonging to hardware independent part.

Reply:
Removed.